googleapiclient/http.py

```diff
@@ -24,34 +24,35 @@
 
 __author__ = 'jcgregorio@google.com (Joe Gregorio)'
 
-import StringIO
+from six import BytesIO, StringIO
+from six.moves.urllib.parse import urlparse, urlunparse, quote, unquote
```

Comment:
I feel `quote` and `unquote` are too short name.
How about `quote as urlquote, unquote as urlunquote` ?